tests/integration/Networking/v2Test.php

```diff
@@ -56,6 +71,9 @@ private function createNetworksAndDelete()
         require_once $path;
 
         foreach($networks as $network) {
+            $this->assertInstanceOf('OpenStack\Networking\v2\Models\Network', $network);
```

Comment:
With these types of assertions, one thing I've started to do is:

```php
$this->assertInstanceOf(Network::class, $network);
```

because my IDE will handle writing out the FQCN for me (as an import at the top of the class). Another reason is that if we ever change the namespace path (which has definitely happened in the past), lots of string are harder to update rather than 1 import path at the top of each test file. I found this out in v1!